nautobot/dcim/forms.py

```diff
@@ -5157,3 +5158,111 @@ class Meta:
             "weight",
             "tags",
         )
+
+
+#
+# Virtual Device Context
+#
+
+
+class VirtualDeviceContextForm(NautobotModelForm):
+    device = DynamicModelChoiceField(
+        queryset=Device.objects.all(),
+    )
+    tenant = DynamicModelChoiceField(
+        queryset=Tenant.objects.all(),
+        required=False,
+    )
+    interfaces = DynamicModelMultipleChoiceField(
+        queryset=Interface.objects.all(), required=False, query_params={"device": "$device"}
+    )
```

Comment:
Should we similarly enhance the Interface edit form to let you specify its assigned VDCs?